server/cpho/views/view_util.py

```diff
@@ -169,6 +171,33 @@ def check_rule(self):
         return test_rule("is_admin_or_hso", self.request.user)
 
 
+def custom_sort_age(ind_data):
+    if ind_data.dimension_type.code == "age":
+        match = re.findall(r"[<]?\d+[+]?", ind_data.literal_dimension_val)
+        if match:
+            first_numeric = match[0]
+            if "<" in first_numeric:
+                return str(
+                    int(first_numeric.replace("<", "").strip()) - 1
+                ).zfill(6)
+            if "+" in first_numeric:
+                return str(
+                    int(first_numeric.replace("+", "").strip()) + 1
+                ).zfill(6)
+            else:
+                return str(int(first_numeric.replace("+", "").strip())).zfill(
+                    6
+                )
+        return ind_data.literal_dimension_val
+
+    else:
+        raise ValueError("Custom Sort: Dimension type is not age")
+
```

Comment:
This is still pretty confusing, can we 

1. Rename this so make it clear this outputs a "sort" score, e.g. `def age_group_sortable_score(indicator_datum)` for a single indicator_datum
2. Remove the dimension_type check, which could cause N+1 problems (we shouldn't need to fetch dimension types on all indicator-datum records, that seems like overkill). The caller should be responsible in making sure they're not passing non-age-group data
3. Write some tests around this function or at least the sort function. Ideally we'd have a bunch of 2-element cases.